template/template.go

```diff
@@ -109,6 +109,39 @@ func NewResolvedValue(value any, options ...ResolvedValueOpt) *ResolvedValue {
 	return r
 }
 
+func newFallbackValue(value any) *ResolvedValue {
```

Comment:
I'd expect it to implement https://github.com/unicode-org/message-format-wg/blob/main/spec/formatting.md#fallback-resolution and nothing else.